robottelo/host_helpers/cli_factory.py

```diff
@@ -241,11 +242,12 @@ def create_object(cli_object, options, values=None, credentials=None):
 }
 
 
-class CLIFactory:
+class CLIFactory(RepositoryHelpers):
     """This class is part of a mixin and not to be used directly. See robottelo.hosts.Satellite"""
 
     def __init__(self, satellite):
         self._satellite = satellite
+        super().__init__()
```

Comment:
If you follow my recommendation in the repository_mixins, then we no longer need the parent class and the init would become this
```suggestion
        self.__dict__.update(get_repo_classes(self._satellite))
```